.coveragerc

```diff
@@ -0,0 +1,10 @@
+[run]
```

Comment:
Can we please drop the hidden (and thus surprising/confusing) `.coveragerc` files and put the configuration in the much more visible `setup.cfg` instead?

Reply:
Nice idea. I am currently working on a more automatic approach to to make the packages more equal.